Zend/zend_language_scanner.l

```diff
@@ -1099,6 +1109,31 @@ NEWLINE ("\r"|"\n"|"\r\n")
 /* compute yyleng before each rule */
 <!*> := yyleng = YYCURSOR - SCNG(yy_text);
 
+<ST_IN_SCRIPTING,ST_LOOKING_FOR_SR_NAME,ST_LOOKING_FOR_VISIBILITY>{WHITESPACE}+ {
+	HANDLE_NEWLINES(yytext, yyleng);
+	return T_WHITESPACE;
+}
+
+<ST_LOOKING_FOR_SR_NAME>"->" {
+	return T_OBJECT_OPERATOR;
+}
+
+<ST_IN_SCRIPTING,ST_LOOKING_FOR_SR_NAME>"namespace"/"\\"|{OPTIONAL_WHITESPACE}"::" {
```

Comment:
What's the second part (optional ws with `::`) for? Also shouldn't there be optional ws before the backslash as well?